package.json

```diff
@@ -131,12 +131,20 @@
       }
     },
     "rules": {
-      "@typescript-eslint/no-unused-vars": [
+      "import/no-anonymous-default-export": [
         1,
         {
-          "vars": "local"
+          "allowObject": true
         }
       ],
+      "react/jsx-pascal-case": [
+        2,
+        {
+          "allowNamespace": true
+        }
+      ],
+      "no-unused-vars": "off",
+      "@typescript-eslint/no-unused-vars": "off",
```

Comment:
In the eslint upgrade, the configuration for this has been moved to `tsconfig.json`.